src/ocl/convolutionocl.cpp

```diff
@@ -2361,6 +2362,8 @@ void ConvolutionDescriptor::FindConvBwdDataAlgorithm(Handle& handle,
             // Implicit GEMM algo
             if(!use_winograd_only)
             {
+                ConvolutionContext params(
```

Comment:
Ditto.

Reply:
@atamazov same as above

Reply:
@atamazov This line also removed. Thanks for your careful review